Projects/Data/Sources/Entities/Post.swift

```diff
@@ -0,0 +1,310 @@
+        pinned = try values.decodeIfPresent(Bool.self, forKey: .pinned)
+        over18 = try values.decodeIfPresent(Bool.self, forKey: .over18)
+        preview = try values.decodeIfPresent(Preview.self, forKey: .preview)
+        mediaOnly = try values.decodeIfPresent(Bool.self, forKey: .mediaOnly)
+        linkFlairTemplateID = try values.decodeIfPresent(String.self, forKey: .linkFlairTemplateID)
+        canGild = try values.decodeIfPresent(Bool.self, forKey: .canGild)
+        spoiler = try values.decodeIfPresent(Bool.self, forKey: .spoiler)
+        locked = try values.decodeIfPresent(Bool.self, forKey: .locked)
+        authorFlairText = try values.decodeIfPresent(String.self, forKey: .authorFlairText)
+        visited = try values.decodeIfPresent(Bool.self, forKey: .visited)
+        numReports = try values.decodeIfPresent(String.self, forKey: .numReports)
+        distinguished = try values.decodeIfPresent(String.self, forKey: .distinguished)
+        subredditID = try values.decodeIfPresent(String.self, forKey: .subredditID)
+        modReasonBy = try values.decodeIfPresent(String.self, forKey: .modReasonBy)
+        removalReason = try values.decodeIfPresent(String.self, forKey: .removalReason)
+        linkFlairBackgroundColor = try values.decodeIfPresent(String.self, forKey: .linkFlairBackgroundColor)
+        id = try values.decodeIfPresent(String.self, forKey: .id)
+        isRobotIndexable = try values.decodeIfPresent(Bool.self, forKey: .isRobotIndexable)
+        reportReasons = try values.decodeIfPresent(String.self, forKey: .reportReasons)
+        author = try values.decodeIfPresent(String.self, forKey: .author)
+        numCrossposts = try values.decodeIfPresent(Int.self, forKey: .numCrossposts)
+        numComments = try values.decodeIfPresent(Int.self, forKey: .numComments)
+        sendReplies = try values.decodeIfPresent(Bool.self, forKey: .sendReplies)
+        whitelistStatus = try values.decodeIfPresent(String.self, forKey: .whitelistStatus)
+        modReports = try values.decodeIfPresent([String].self, forKey: .modReports)
+        authorPatreonFlair = try values.decodeIfPresent(Bool.self, forKey: .authorPatreonFlair)
+        authorFlairTextColor = try values.decodeIfPresent(String.self, forKey: .authorFlairTextColor)
+        permalink = try values.decodeIfPresent(String.self, forKey: .permalink)
+        parentWhitelistStatus = try values.decodeIfPresent(String.self, forKey: .parentWhitelistStatus)
+        stickied = try values.decodeIfPresent(Bool.self, forKey: .stickied)
+        url = try values.decodeIfPresent(String.self, forKey: .url)
+        subredditSubscribers = try values.decodeIfPresent(Int.self, forKey: .subredditSubscribers)
+        createdUTC = try values.decodeIfPresent(Int.self, forKey: .createdUTC)
+        media = try values.decodeIfPresent(Media.self, forKey: .media)
+        isVideo = try values.decodeIfPresent(Bool.self, forKey: .isVideo)
+    }
+    // swiftlint:enable identifier_name
```

Comment:
Why is this here? It is enabled by default anyway.

Reply:
Fixed